app/code/Scandiweb_Test/Setup/Patch/Data/Test.php

```diff
@@ -1,12 +1,13 @@
 <?php
 /**
- * @category    Example
- * @package     Example_Migration
- * @author      Ralfs Aizsils <info@scandiweb.com>
- * @copyright   Copyright (c) 2021 Scandiweb, Ltd (https://scandiweb.com)
+ * @category Scandiweb
+ * @package Scandiweb_ScandiwebTest
+ * @author Wiktor Koscielny <info@scandiweb.com>
+ * @copyright Copyright (c) 2022 Scandiweb, Inc (https://scandiweb.com)
+ * @license http://opensource.org/licenses/OSL-3.0 The Open Software License 3.0 (OSL-3.0)
  */
 
-namespace Scandiweb\ScandiwebTest\Setup\Patch\Data;
```

Comment:
You need to set the namespace for the class, please replace `namespace Scandiweb\ScandiwebTest\Setup\Patch\Data;` after `declare (strict_types=1);`